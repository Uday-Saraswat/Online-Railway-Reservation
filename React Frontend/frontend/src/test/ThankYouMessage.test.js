import React from 'react';
import renderer from 'react-test-renderer';
import ThankYouMessage from '../components/ThankYouMessage'

const mockedUsedNavigate = jest.fn();
jest.mock('react-router-dom', () => ({
    ...jest.requireActual('react-router-dom'), Link: () => mockedUsedNavigate,

}));

afterAll(() => {
    jest.clearAllMocks();
});

it('renders correctly', () => {
    const tree = renderer
        .create(<ThankYouMessage/>)
        .toJSON();
    console.log(tree)

    expect(tree).toMatchSnapshot();

});