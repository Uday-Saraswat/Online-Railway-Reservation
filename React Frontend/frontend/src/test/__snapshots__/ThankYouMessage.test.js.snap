// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <center>
    <img
      src="back7edit.jpg"
      width="1000"
    />
  </center>
</div>
`;
